spec/models/student_work_group_spec.rb

```diff
@@ -54,5 +54,11 @@
       subject.user = create(:teacher)
       expect(subject).not_to be_valid
     end
+
+    it 'with more than one turn set to true' do
+      create(:work_group)
```

Comment:
no need for this line, when you create a student_work_group it should automatically create a work gorup

Reply:
If I don't use that I get  ActiveRecord::RecordInvalid:
       Validation failed: Work group must exist, Work group can't be blank

Reply:
you mean if you write like
```
      create(:student_work_group, turn: true)
      expect { create(:student_work_group, work_group: WorkGroup.last, turn: true)}.to raise_error(ActiveRecord::RecordInvalid)
```
?

Reply:
if you leave with the `work_group: WorkGroup.last` you'll get the error becausethere are no workgroups, but if you remove that part it will create a workgroup automatically

Reply:
Sorry I might have left WorkGroup.last on the first line that's why it was failing. It works without it now.